### PR TITLE
Resolved issue #4.

### PR DESCRIPTION
**Pull request by HaoHuynh0301**
Please check it for me. Thank you !

### CODE REVIEW

File: src/Routes.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Suspense } from "react";
 import { Route, Switch } from "react-router-dom";
+import { Sidebar } from './components/Sidebar';
```
Comment (jakezachariahnixon) [inline src/Routes.tsx]: This import hasn't been used in the file, it should be removed

File: src/components/Sidebar/UserDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    globalCss
+} from '../../styles/stitches.config';
+import userIcon from './img/userIcon.png';
+
+type CardProps = {
+    name: string,
+}
+
+const Avatar = () => <aside>
+    <div style = {{
```
Comment (jakezachariahnixon) [inline src/components/Sidebar/UserDetails.tsx]: Can you write these using stitches?
Docs: https://stitches.dev/
Example: https://github.com/jakezachariahnixon/todo-app/tree/main/src/components/Badge

**Review by jakezachariahnixon — CHANGES_REQUESTED** [review]
Nice work, a few comments:

Your new component should be in a new directory under components, e.g: components/UserDetails

Please can you check that fonts and sizings line up with the designs: https://www.figma.com/file/x995jwsHZtMlKND63EodmO/Todo-App?node-id=0%3A1

File: src/components/Sidebar/UserDetails.tsx (hunk 2 above)
Comment (HaoHuynh0301) [reply src/components/Sidebar/UserDetails.tsx]: Please check it for me. Thank you ! @jakezachariahnixon

**Comment by jakezachariahnixon** [comment]
Looking good. Before merging can we add a story and a snapshot test?
- Create a story file in the UserDetails directory so we can see the component inside Storybook
  - You might want to add a UserDetailsProps interface to pass to the story, this can be empty for now: 
  - `export interface UserDetailsProps {}`
- Add a snapshot test inside the UserDetails directory so we can make sure we don't make any accidental changes in future

You can use the Badge component as reference if needed.

**Comment by HaoHuynh0301** [comment]
I've edited on demand. Please check it for me. Thank you !

**Comment by jakezachariahnixon** [comment]
Thanks for the edits. 

The Storybook shows that there is an accessibility violation (see the accessibility tab at the bottom of the Storybook). We need to add an alt tag to the img.

Once this is fixed then I will be happy to merge!

**Review by jakezachariahnixon — APPROVED** [review]

**Pull request merged**